main.rb

```diff
@@ -0,0 +1,131 @@
+require './app'
+
+def welcome
+  puts 'Welcome to the School Library App'
+end
+
+def create_person(library)
+  print 'Do you want to create a Student (1) or a teacher (2)? [Input the number]:'
+
+  input = gets.chomp.to_i
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Age: '
+  age = gets.chomp.to_i
+
+  if input == 1
+    print 'Has parent permission? y/n :'
+    permission = gets.chomp
+
+    library.create_student(age, name, permission == 'y')
+  else
+    print 'Specialization:'
+    spec = gets.chomp
+
+    library.create_teacher(age, name, spec)
+  end
+
+  puts input == 1 ? 'Student created Successfuly!' : 'Teacher created Successfuly!'
+
+  user_choice(library)
+end
+
+def create_book(library)
+  puts 'Enter book Title:'
+  title = gets.chomp
+  puts 'Enter authors name'
+  author = gets.chomp
+  library.add_book(title, author)
+
+  puts 'Book created successfully!'
+  user_choice(library)
+end
+
+def create_rental(library)
+  puts 'Select a book from the book list below by number: '
+  library.list_books
+
+  book_id = gets.chomp.to_i
+
+  puts 'Select a person from the following list by number (not ID): '
+  library.list_people
+
+  person_id = gets.chomp.to_i
+
+  print 'Enter Date: '
+  date = gets.chomp
+
+  library.create_rental(book_id - 1, person_id, date)
+
+  puts 'Rental created Successfully!'
+  user_choice(library)
+end
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure the user can be able to create the `rental` successfully. Kindly make sure to fix the issue.

    - ### THE ERROR BEING DISPLAYED WHEN WE TRY TO CREATE RENTAL
    
       ![image](https://user-images.githubusercontent.com/87197412/233805920-09dd0d35-9b45-46c6-8b91-e969a755fba2.png)